src/actions/actionExport.tsx

```diff
@@ -214,20 +215,20 @@ export const actionExportWithDarkMode = register({
     };
   },
   PanelComponent: ({ appState, updateData }) => (
-    <label
+    <div
       style={{
         display: "flex",
         justifyContent: "flex-end",
-        marginTop: "-32px",
+        marginTop: "-45px",
         marginBottom: "10px",
       }}
     >
-      <input
-        type="checkbox"
-        checked={appState.exportWithDarkMode}
-        onChange={(event) => updateData(event.target.checked)}
-      />{" "}
-      {t("labels.exportWithDarkMode")}
-    </label>
+      <DarkModeToggle
```

Comment:
Let's make the `DarkModeToggle` component take `props.title` so we can supply a different tooltip for the export.

Reply:
What should we pass as title? "Export with dark mode" or "Export with light mode" depending on state?

Reply:
Maybe "Toggle export color scheme" for both is enough?

Reply:
For some reason, when passing the title this way https://github.com/excalidraw/excalidraw/pull/3046/commits/da8a587faf1e8083cbb1d3f80c8b16fdd2a7d6b0 somehow the other instance where we use this component to change background overrides title.

Reply:
Should work now!